editor/src/components/canvas/controls/grid-controls.tsx

```diff
@@ -842,16 +863,16 @@ export const GridControls = controlForStrategyMemoized<GridControlsProps>(({ tar
                     id={id}
                     data-testid={id}
                     style={{
-                      borderTop: `1px solid ${borderColor}`,
-                      borderLeft: `1px solid ${borderColor}`,
+                      borderTop: `3px solid ${borderColor}`,
```

Comment:
ain't this a bit too thick now?

Reply:
https://github.com/concrete-utopia/utopia/pull/6301/commits/afd194782dea6df86984ac15175197711b47c61d